docs/src/pages/docs/functions/combinators.md

```diff
@@ -64,6 +64,119 @@ safeYell(42)
 //=> Left 42
 ```
 
+#### compose2
+
+`crocks/combinators/compose2`
+
+```haskell
+compose2 :: (c -> d -> e) -> (a -> c) -> (b -> d) -> a -> b -> e
+```
+
+`compose2` allows for composition between a `binary` function and
+two `unary` functions. `compose2` takes a `binary` function followed by
+two `unary` functions and returns a `binary` function that maps the first
+argument with the first `unary` and the second with the second, passing
+the results to the given `binary` and returning the result.
+
+```javascript
+import compose2 from 'crocks/combinators/compose2'
+
+import and from 'crocks/logic/and'
+import applyTo from 'crocks/combinators/applyTo'
+import flip from 'crocks/combinators/flip'
+import hasProp from 'crocks/predicates/hasProp'
+import isNumber from 'crocks/predicates/isNumber'
+import liftA2 from 'crocks/helpers/liftA2'
+import map from 'crocks/pointfree/map'
+import prop from 'crocks/Maybe/prop'
+import safe from 'crocks/Maybe/safe'
+import safeLift from 'crocks/Maybe/safeLift'
+
+// isNonZero :: Number -> Boolean
+const isNonZero = x => x !== 0
+
+// isValidDivisor :: Number -> Boolean
+const isValidDivisor = and(isNumber, isNonZero)
+
+// divideBy :: Number -> Number -> Number
+const divideBy = x => y => y / x
+
+// safeDivide :: Number -> Number -> Maybe Number
+const safeDivide =
+  compose2(liftA2(divideBy), safe(isValidDivisor), safe(isNumber))
+
+safeDivide(0.5, 21)
+//=> Just 42
+
+safeDivide('0.5', 21)
+//=> Nothing
+
+safeDivide(0.5, '21')
+//=> Nothing
+
+safeDivide(29, 0)
+//=> Just 0
+
+safeDivide(0, 29)
+//=> Nothing
+
+// Item :: { id: Integer }
+// Items :: Array Item
+const items =
+  [ { id: 2 }, { id: 1 } ]
+
+// pluck :: String -> Array Object -> Maybe a
+const pluck =
+  compose2(applyTo, prop, flip(map))
```

Comment:
This is to show usage in relation to https://github.com/evilsoft/crocks/issues/338